templates/todo/api/java/src/main/java/com/microsoft/azure/simpletodo/configuration/WebConfiguration.java

```diff
@@ -21,7 +21,17 @@ public WebMvcConfigurer webConfigurer() {
         return new WebMvcConfigurer() {
             @Override
             public void addCorsMappings(CorsRegistry registry) {
-                registry.addMapping("/**").allowedOrigins("*").allowedMethods("*").allowedHeaders("*");
+                var apiUrl = System.getProperty("REACT_APP_WEB_BASE_URL");
+
+                if (apiUrl != "") {
+                    registry.addMapping("/**").allowedOrigins("https://portal.azure.com",
+                            "https://ms.portal.azure.com",
+                            apiUrl).allowedMethods("*").allowedHeaders("*");
+                } else {
+                    registry.addMapping("/**").allowedOrigins("*").allowedMethods("*").allowedHeaders("*");
+                    System.out.println("error: could get environment variable REACT_APP_WEB_BASE_URL");
```

Comment:
I would change this. Instead of `error`, I would just print something like:
"Setting CORS = * (allow all origins) because env var `REACT_APP_WEB_BASE_URL` has not a value or is not set."